app/api/endpoint.py

```
@@ -78,4 +79,13 @@ def recommend_on_demand(request_body: UserRequest):
     except Exception as e:
         logger.error("추천 생성 중 오류 발생", exc_info=True)
-        raise HTTPException(status_code=500, detail="내부 서버 오류가 발생했습니다.") from e
\ No newline at end of file
+        raise HTTPException(status_code=500, detail="내부 서버 오류가 발생했습니다.") from e
+
+@router.post("/trigger-batch")
+def trigger_batch():
+    try:
+        run_batch()
+        return JSONResponse(status_code=200, content={"message": "Batch recommendation process executed successfully."})
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)